README.md

```diff
@@ -63,6 +63,8 @@ Edit `config.json` and rerun to output your customized stylesheet.
 
 The configuration for the fluid typographic scale. Affects font size classes and the respective custom properties referenced by those classes.
 
+**Note: Enhance Styles will automatically assign the generated base step in this scale (`var(--text0)`) to the `body` font size.**
```

Comment:
@kristoferjoseph Thoughts?

Reply:
I would add "so there is not need to explicitly set a font size on the body"